src/Stratis.Bitcoin/Connection/ConnectionManager.cs

```diff
@@ -7,22 +7,29 @@
 using Microsoft.Extensions.Logging;
 using NBitcoin;
 using NBitcoin.Protocol;
-using NBitcoin.Protocol.Behaviors;
 using Stratis.Bitcoin.Base;
 using Stratis.Bitcoin.BlockPulling;
 using Stratis.Bitcoin.Configuration;
 using Stratis.Bitcoin.Configuration.Logging;
 using Stratis.Bitcoin.Configuration.Settings;
+using Stratis.Bitcoin.P2P;
 using Stratis.Bitcoin.Utilities;
 
 namespace Stratis.Bitcoin.Connection
 {
     public interface IConnectionManager : IDisposable
     {
-        NodesGroup AddNodeNodeGroup { get; }
+        /// <summary> The <see cref="PeerConnector"/> that is used when we use the -addnode argument when running the node.</summary>
```

Comment:
`The PeerConnector` is actually redundant information, everyone sees that type is PeerConnector. Maybe little explanation of this actually is would be better than mentioning the type again.

consider enclosing `-addnode` to "" like `"-addnode"`